README.md

```diff
@@ -1,181 +1,196 @@
+# LTI 1.3 Advantage Library
 
```

Comment:
Briefly skimmed through the README I'm not sure I understand what all this library offers to help get an app LTI 1.3 compatible. It's okay if the answer is no, but is there anything that can add to the overview to make the benefits of this package clearer at first glance?